sass/selectors/_forms.scss

```diff
@@ -179,3 +179,49 @@ form button {
 .autocomplete-dropdown {
   @include autocomplete-dropdown;
 }
+
+// Demo Form Layouts
+.horizontal-flip {
+  flex-direction: row-reverse;
+  .demo-form__wrapper {
```

Comment:
I'm going to approve this, as this is fine. But we should pull the demo-form classes out in another refactor as they should be in Employer.

Reply:
you mean in the style-base? I might as well do it now.

Reply:
`horizontal-flip` and `vertical-flip` can remain, but the `demo-form__` classes can go to Employer since those are specific to `sections/_demo.scss`.

So in `demo.scss` you'd have:
```
.horizontal-flip {
  .demo-form__wrapper { ... }
}
```
as an example.

Reply:
Here would you would just have:

```
horizontal-flip { ... }
```